RtEthSample/txqueue.cpp

```diff
@@ -118,6 +70,17 @@ RtGetPacketChecksumSetting(
         // Prioritize layer4 checksum first
         if (checksumInfo->Layer4 == NetPacketTxChecksumActionRequired)
         {
+
+            const USHORT layer4HeaderOffset =
+                packet->Layout.Layer2HeaderLength +
+                packet->Layout.Layer3HeaderLength;
+
+            UNREFERENCED_PARAMETER(layer4HeaderOffset);

```

Comment:
Added this here because when building in release mode layer4HeaderOffset does not get initialized and this was causing a warning. Not sure if there is a better way to deal with it.